Snowdog_Components/components/Molecules/form/checkbox/_checkbox.scss

```
@@ -24,6 +24,9 @@
 
         .checkbox__label {
             color: $checkbox__label-color--active;
+            &:after {
```

there should be an empty line between sectors on the same level:
```suggestion
            color: $checkbox__label-color--active;
            
            &:after {
```